src/include/hip/rpp_hip_common.hpp

```diff
@@ -88,6 +88,9 @@ struct RPPTensorFunctionMetaData
 #define ONE_OVER_255                    0.00392157f
 #define ONE_OVER_256                    0.00390625f
 #define SIX_OVER_360                    0.01666667f
+#define ONE_OVER_6                      0.1666666f
```

Comment:
I think too many of these definitions can cause rounding errors. Please use the following instead
const float ONE_OVER_SIX = 1.0f/6;
That way compiler will only do the calculation once and keep it as constant.

Reply:
Done